dan/webserver/webserver.py

```diff
@@ -5,10 +5,10 @@
 
 
 app = Microdot()
-
+MAX_PINNED_MESSAGES = 10
```

Comment:
Added MAX_PINNED_MESSAGES and MAX_MESSAGE_HISTORY consts for max number of messages.